packages/debugger/lib/web3/adapter.js

```diff
@@ -104,4 +122,27 @@ export default class Web3Adapter {
     let code = await this.web3.eth.getCode(address, block);
     return code === "0x0" ? "0x" : code;
   }
+
+  async reverseEnsResolve(address) {
+    if (!this.ens) {
+      debug("no ens you fool!");
```

Comment:
i have a unique appreciation for the kind of software that insults those who diagnose its internal behavior